plugins/repository-s3/src/internalClusterTest/java/org/opensearch/repositories/s3/S3BlobStoreRepositoryTests.java

```diff
@@ -81,11 +80,8 @@ public void setUp() throws Exception {
         if (randomBoolean()) {
             region = "test-region";
         }
-        if (region != null && randomBoolean()) {
-            signerOverride = randomFrom("AWS3SignerType", "AWS4SignerType");
-        } else if (randomBoolean()) {
-            signerOverride = "AWS3SignerType";
-        }
+        signerOverride = AwsRequestSigner.VERSION_FOUR_SIGNER.getName();
+        SocketAccess.doPrivileged(() -> System.setProperty("opensearch.path.conf", "config"));
```

Comment:
Same comment as for discovery plugin, we should better reset it back once the test case finishes